### DISCUSSION TITLE
Try J: a website for running J code (similar to repl.it, etc.)

### DISCUSSION THREAD

**Discussion by nathanesau** [codereport/jsource#130] [Show and tell]
Hi All,

I created a site for running J code in your browser. This should make it easy for anyone to try out J, without going to work of installing and building latest J.

Here is a link to the site (Digital Ocean).

http://tryj.freeddns.org/

Here is a quick demo.

![try_j](https://user-images.githubusercontent.com/4649987/107161044-fcdfb780-6967-11eb-8abc-1d86caa7b4b9.gif)

The source code is here: https://github.com/nathanesau/try_j

This took about 4 hours. It is made using Flask. Obviously there is more that can be done. :smile:

Let me know what you think!

**Comment by codereport** [comment]
This is awesome! Thanks!
